Algorithm-Drop-it.md

```diff
@@ -69,7 +69,7 @@ drop([1, 2, 3], function(n) {return n < 3; });
 ```
 
 # Code Explanation
-- Use a while loop with `Array.shift()` to continue checking and dropping the first element of the array until the function returns true.
+- Use a while loop with `Array.shift()` to continue checking and dropping the first element of the array until the function returns true. It also makes sure teh array is not empty first to avoid infinitie loops

```

Comment:
- typo `teh` -> `the`
- and i don't want to avoid `infinitie` loops ;) `infinitie` -> `infinity`
- and it's `Array.prototype.shift` method or simply `arr.shift()`
- end the sentence with a full stop.